examples/python/images/image_reader.py

```diff
@@ -0,0 +1,68 @@
+###############################################################################
+# (c) 2005-2015 Copyright, Real-Time Innovations.  All rights reserved.       #
+# No duplications, whole or partial, manual or electronic, may be made        #
+# without express written permission.  Any such copies, or revisions thereof, #
+# must display this notice unaltered.                                         #
+# This code contains trade secrets of Real-Time Innovations, Inc.             #
+###############################################################################
+
+"""Reader without using the wait call."""
+
+from __future__ import print_function
+from sys import path as sys_path
+from os import path as os_path
+import matplotlib.pyplot as plot
+import matplotlib.animation as animation
+import numpy
+
+# Updating the system path is not required if you have pip-installed
+# rticonnextdds-connector
+file_path = os_path.dirname(os_path.realpath(__file__))
+sys_path.append(file_path + "/../../../")
+import rticonnextdds_connector as rti
+
+
```

Comment:
```
(python2_env) sam@rti-10678:~/working/trees/connector/connector-py$ python examples/python/images/image_reader.py 
Traceback (most recent call last):
  File "examples/python/images/image_reader.py", line 14, in <module>
    import matplotlib.pyplot as plot
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/pyplot.py", line 115, in <module>
    _backend_mod, new_figure_manager, draw_if_interactive, _show = pylab_setup()
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/backends/__init__.py", line 63, in pylab_setup
    [backend_name], 0)
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/backends/backend_tkagg.py", line 4, in <module>
    from . import tkagg  # Paint image to Tk photo blitter extension.
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/matplotlib/backends/tkagg.py", line 5, in <module>
    from six.moves import tkinter as Tk
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/six.py", line 203, in load_module
    mod = mod._resolve()
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/six.py", line 115, in _resolve
    return _import_module(self.mod)
  File "/home/sam/working/trees/connector/python2_env/lib/python2.7/site-packages/six.py", line 82, in _import_module
    __import__(name)
  File "/usr/local/lib/python2.7/lib-tk/Tkinter.py", line 39, in <module>
    import _tkinter # If this fails your Python may not be configured for Tk
ImportError: No module named _tkinter
```

Reply:
But the writer works

Reply:
Running on python3 gives the following error too:
```
sam@rti-10678:~/working/trees/connector/connector-py/examples$ python3 python/images/image_reader.py 
python/images/image_reader.py:67: UserWarning: Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.
  plot.show()
Exiting...
```

Reply:
I've tested it on Mac and windows. I'll try linux.

Reply:
The writer doesn't use matplotlib

Reply:
I've added image_reader_file.py for systems that don't have a GUI backend by default.